apps/dashboard/src/app/nebula-app/(app)/components/ChatPageContent.tsx

```diff
@@ -118,10 +134,13 @@ export function ChatPageContent(props: {
             walletAddress: null,
           };
 
+      // Only set wallet address from connected wallet
       updatedContextFilters.walletAddress = address || null;
-      updatedContextFilters.chainIds = activeChain
-        ? [activeChain.id.toString()]
-        : [];
+      
+      // Only set chain if we don't already have chains (either from cache or session)
+      if (!updatedContextFilters.chainIds?.length) {
+        updatedContextFilters.chainIds = activeChain ? [activeChain.id.toString()] : ['1'];
```

Comment:
This check `!updatedContextFilters.chainIds?.length` makes it so that whenever user updates the chain before starting out a session - it is longer set in the context